Source/K8sMock/namespaces/NamespacesController.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { Controller, Get, Route, Tags } from 'tsoa';
+import { PodList } from '@shared/backend/k8s/PodList';
+
+const namespaces = require('./namespaces.json') as PodList;
+
+@Route('api/v1/namespaces')
+@Tags('core_v1')
+export class NamespacesController extends Controller {
+
+    @Get()
+    async getNamespaces(): Promise<PodList> {
+        return namespaces;
+    }
+}
```

Comment:
I presume `PodList` references here, should be `NamespaceList`?

Reply:
:) Indeed.. Thnx.. Will fix..